bears/vcs/git/GitCommitBear.py

```diff
@@ -66,6 +99,7 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
             self.err('git:', repr(stderr))
             return
 
+        body = stdout.rstrip('\n')
```

Comment:
you should move the commit introducing the `shortlog` and `body` variables before this commit and use them accordingly here^^